.circleci/config.yml

```diff
@@ -142,6 +142,9 @@ jobs:
       - install_chrome
       - attach_workspace:
           at: /home/circleci/
+      - run:
+          name: Setup python by installing protobuf and enum
```

Comment:
Nit here & below: set up (verb form is more appropriate here).

Reply:
Done.